style.css

```diff
@@ -343,9 +343,159 @@ section#grave-care {
 
 .grave-care-overlay {
   background-color: rgba(0, 0, 0, 0.72);
-  position: absolute;
   width: 100%;
   height: 100%;
   padding: 5rem 15rem;
   text-align: center;
 }
+
+/*---Section Documents---*/
+
+section#speaker {
+  background-color: #fff;
+  display: flex;
+  padding: 60px 0px;
```

Comment:
Maybe we could use rems here as well? :)

Reply:
Done :)